### ISSUE TITLE
Coverage on the new / modifies files

### ISSUE THREAD

**Issue by pavelfomin** [craigatk/projektor#927]
One of the things that I miss from Sonar is a specific section of coverage on the new / modifies files.
This can be very helpful for the projects that have low code coverage to get the coverage gradually up on every PR.

**Comment by craigatk** [comment]
Thanks for the suggestion! I agree, it'd be great to have code coverage on just the changes in a PR. Projektor already has access to the pull request to make PR comments, I wonder if I can pull the changes from the PR as well.

**Comment by pavelfomin** [comment]
I've also noticed that the coverage does report an increase / decrease. Does Projektor compare the coverage with a previous build on the same branch?
 
<img width="1251" alt="image" src="https://user-images.githubusercontent.com/2952816/208958578-2de1f3fa-55be-4b2b-a530-ccf402b915ad.png">

**Comment by craigatk** [comment]
Good catch on the Projektor report coverage increase / decrease - that's the difference in coverage between the branch and the mainline codebase branch.
